AppCenter/AppCenter/Internals/Model/CommonSchema/MSLocExtension.h

```diff
@@ -11,6 +11,6 @@
 /**
  * Time zone on the device.
  */
-@property(nonatomic, copy) NSString *timezone;
+@property(nonatomic, copy) NSString *tz;
```

Comment:
The shortened name of the property seems a little too short for me. I didn't make the connection from `tz` to `timezone`.

Reply:
@TroubleMakerBen [I just leave it here](http://man7.org/linux/man-pages/man3/tzset.3.html)

Reply:
Just matching the attribute name in the schema with the property as much as possible, it's consistent with the Android AC SDK.

Reply:
understood!